documentation/modules/overview/con-configuration-points-broker.adoc

```diff
@@ -43,6 +43,8 @@ External listeners expose Kafka by specifying a `type`:
 * `ingress` to use Kubernetes _Ingress_ and the {NginxIngressController}.
 --
 
+If you are using xref:security-configuration-authentication_{context}[{oauth} for token-based authentication], you can configure listeners to connect to the authorization server.  
```

Comment:
This is a bit weird ... _configure listeners to connect to_. I think we should try to find something better. MAybe _to **use** the authentication server_?